enterprise/server/backends/distributed/distributed.go

```diff
@@ -401,7 +406,7 @@ func (c *Cache) getLookasideEntry(r *rspb.ResourceName) ([]byte, bool) {
 	found := false
 	entry, ok := c.lookaside.Get(k)
 	if ok {
-		if time.Since(time.UnixMilli(entry.createdAtMillis)) > *lookasideCacheTTL {
+		if *lookasideCacheTTL > 0*time.Second && time.Since(time.UnixMilli(entry.createdAtMillis)) > *lookasideCacheTTL {
```

Comment:
does just 0 work here instead of 0*time.Second?

Reply:
Hmm, thought it wouldn't, but it does seem to. Dropped the `time.Second` here and below.